modules/core/src/main/scala/org/scalasteward/core/nurture/NurtureAlg.scala

```diff
@@ -153,10 +154,11 @@ final class NurtureAlg[F[_]](
         .traverse(vcsExtraAlg.getReleaseRelatedUrls(_, data.update))
       branchName = vcs.createBranch(config.vcsType, data.fork, data.update)
       migrations = migrationAlg.findMigrations(data.update)
+      existingArtifactUrls <- artifactIdToUrl.toList.filterA(a => existenceClient.exists(a._2))
```

Comment:
I think we shouldn't filter out the whole artifact when url is broken. We can still write the artifact id in PR body but as regular text instead of a url.

Reply:
When there is no URL for an `artifactId` we write the `artifactId` without the URL:
https://github.com/fthomas/scala-steward/blob/master/modules/core/src/main/scala/org/scalasteward/core/vcs/data/NewPullRequestData.scala#L121-L124

Reply:
Oh i see, i wasn't aware of that 👍